deployments/helm-chart/templates/controller-globalconfiguration.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if .Values.controller.globalConfiguration.create }}
+apiVersion: k8s.nginx.org/v1alpha1
+kind: GlobalConfiguration
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
would it make sense to use `nginx-ingress.name` defined in _helpers.tpl. ?  this way the resource will have the same name as the other  resources (or the prefixes of the other resources)